features/step_definitions/networking.rb

```diff
@@ -690,11 +696,22 @@
 Given /^I restart the ovs pod on the#{OPT_QUOTED} node$/ do | node_name |
   ensure_admin_tagged
   ensure_destructive_tagged
-
-  ovs_pod = BushSlicer::Pod.get_labeled("app=ovs", project: project("openshift-sdn", switch: false), user: admin) { |pod, hash|
-    pod.node_name == node_name
-  }.first
-  @result = ovs_pod.ensure_deleted(user: admin)
+  node = node(node_name)
```

Comment:
@anuragthehatter you put `#{OPT_QUOTED}`...I think this call will fail if `node_name` is `nil`

Reply:
Fixed it. Thanks